packages/core/src/new-api/internal/execution/transactions/chain-dispatcher.ts

```diff
@@ -48,16 +56,25 @@ export class EthersChainDispatcher implements ChainDispatcher {
 
       tx.gasPrice = gasPrice;
     }
+    try {
+      const response = await signer.sendTransaction(tx);
 
-    const response = await signer.sendTransaction(tx);
+      const txHash = response.hash;
 
-    const txHash = response.hash;
+      const receipt = await this._transactionProvider.wait(txHash);
```

Comment:
Just to confirm: we won't use this behavior (i.e. `.wait`), right?

Reply:
This will all be restructured in the monitoring PR.